.github/workflows/models.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js Models
+
+on: [ workflow_dispatch, pull_request ]
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    name: Generate Models
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Generate models
+        run: make models
+      - name: Overwrite current models  
+        run: |
+          rm -rf src/typings/checkout
+          cp -r build/model src/typings/checkout
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
```

Comment:
who is peter-evans and is he legit?

Reply:
I've found this step on the marketplace https://github.com/marketplace/actions/create-pull-request, it has around 1k stars. The author also has a few other actions https://github.com/marketplace?type=actions&query=publisher%3Apeter-evans+.

Reply:
Seems legit then. Let's go with it for now